charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -156,6 +157,44 @@ spec:
                         serviceAccountKeyJSON:
                           type: string
                           minLength: 1
+                    s3Compatible:
+                      type: object
+                      required:
+                        - bucket
+                        - endpoint
+                      oneOf:
+                      - required:
+                        - accessKeyID
+                        - secretAccessKey
+                        allOf:
+                        - not:
+                            required:
+                            - secretName
+                      - required:
+                        - secretName
+                        allOf:
+                        - not:
+                            required:
+                            - accessKeyID
+                            - secretAccessKey
+                      properties:
+                        bucket:
+                          type: string
+                          minLength: 1
+                        secretName:
+                          type: string
+                          minLength: 1
+                        createSecret:
+                          type: boolean
+                        accessKeyID:
+                          type: string
+                          minLength: 1
+                        secretAccessKey:
+                          type: string
+                          minLength: 1
+                        endpoint:
```

Comment:
```yaml
endpoint:
  type: object
  format: uri
  minLength: 1
```

Reply:
@timflannagan1 this is definitely `type: string` not `object`